packages/tests/server/react/invalidateQueries.test.tsx

```diff
@@ -216,18 +217,17 @@ describe('invalidateQueries()', () => {
       expect(utils.container).toHaveTextContent('count:test:0');
     });
 
-    for (const testId of [
+    for (const [index, testId] of [
       'invalidate-1-string',
       'invalidate-2-exact',
       'invalidate-3-all',
       'invalidate-4-predicate',
-    ]) {
+    ].entries()) {
+      await userEvent.click(utils.getByTestId(testId));
       await waitFor(async () => {
         // click button to invalidate
-        await userEvent.click(utils.getByTestId(testId));
-
         // should become stale straight after the click
-        expect(utils.container).toHaveTextContent(`count:test:1`);
+        expect(utils.container).toHaveTextContent(`count:test:${index + 1}`);
```

Comment:
yea that seemed strange to me but i didn't wanna modify it since it had worked before xddd

i guess these warnings do shows some flaws